conda-env/ci.yml

```
@@ -8,21 +8,21 @@ dependencies:
     # Base
     # ==================
     # NOTE: If versions are updated, also `additional_dependencies` list for mypy in `.pre-commit-config.yaml`
-    - python=3.10.10
-    - pip=23.1.2
-    - numpy >=2.0.0,<2.1.0
+    - python >=3.10,<3.14
+    - pip >=23.1.2
+    - numpy >=2.0.0,<3
     - cartopy >=0.22.0
     - matplotlib >=3.7.1
-    - cdp=1.7.0
-    - eofs=2.0.0
-    - seaborn=0.12.2
-    # - enso_metrics=1.1.5
+    - cdp >=1.7.0
+    - eofs >=2.0.0
+    - seaborn >=0.12.2
+    #- enso_metrics >=1.1.5
     - xcdat >=0.7.3
```

```suggestion
    - xcdat >=0.10.0
```

I suggest constraining xCDAT to the latest version, as pcmdi_metrics relies on xCDAT's regridder (especially regrid2). The latest version contains the most recent bug fixes and enhancements.